<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>597</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>806</width>
    <height>597</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>806</width>
    <height>597</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>331</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1、把文档或在线文档导出成csv，选择文件-打开&lt;/p&gt;&lt;p&gt;2、选择检查模式，配置参数&lt;/p&gt;&lt;p&gt;3、点击检查&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>设置：</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>150</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>检查尖括号中的空格</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>需要检查是否匹配的内容：</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="txt_match">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>181</width>
        <height>191</height>
       </rect>
      </property>
      <property name="plainText">
       <string>{}
&lt;&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>需要排除的对象名称：</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="txt_exclude">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>181</width>
        <height>191</height>
       </rect>
      </property>
      <property name="plainText">
       <string>KeyCode</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen"/>
   </widget>
   <widget class="QMenu" name="menumode">
    <property name="title">
     <string>模式</string>
    </property>
    <addaction name="actiontranslate"/>
   </widget>
   <widget class="QMenu" name="menucheck">
    <property name="title">
     <string>检查</string>
    </property>
    <addaction name="actionruncheck"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menumode"/>
   <addaction name="menucheck"/>
  </widget>
  <action name="actionopen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actiontranslate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>海外翻译检查</string>
   </property>
  </action>
  <action name="actionruncheck">
   <property name="text">
    <string>执行检查</string>
   </property>
  </action>
  <action name="action2333">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2333</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
